### PR TITLE
DAOS-11749 test: Use default svcn in /pool/create_capacity.py test

### PR DESCRIPTION
**Pull request by phender**
Update the /pool/create_capacity.py test to use the default pool svcn to avoid overutilizing engine 0, resulting in pool destroy timeouts.

Also fixing a pylint error in harness/advanced.py.

Quick-functional: true
Test-tag: create_performance
Test-repeat: 5

Required-githooks: true

Signed-off-by: Phil Henderson <phillip.henderson@intel.com>

### CODE REVIEW

**Review by daltonbohning — APPROVED** [review]

File: src/tests/ftest/pool/create_capacity.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/python3
```
Comment (daltonbohning) [inline src/tests/ftest/pool/create_capacity.py:1]: If you need to push again - can be removed

**Review by daltonbohning — APPROVED** [review]

**Review by shimizukko — DISMISSED** [review]

File: src/tests/ftest/pool/create_capacity.py (hunk 1 above)
Comment (phender) [reply src/tests/ftest/pool/create_capacity.py]: Done

**Review by shimizukko — APPROVED** [review]

**Review by daltonbohning — APPROVED** [review]

**Comment by phender** [comment]
All five repetitions of the pool/create_capacity.py test passed in https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-10455/3

**Review by jolivier23 — COMMENTED** [review]
Quick-functional should not be used for landings.  Does this pass bandit?

**Comment by phender** [comment]
> Quick-functional should not be used for landings. Does this pass bandit?

It doesn't make any changes that would fail the Bandit check.  If we need, though, I push another commit.

**Pull request merged**